lib/ret/hub.ex

```diff
@@ -31,6 +31,8 @@ defmodule Ret.Hub do
     field(:entry_code_expires_at, :utc_datetime)
     field(:last_active_at, :utc_datetime)
     field(:creator_assignment_token, :string)
+    field(:embed_token, :string)
+    field(:embedded, :boolean)
```

Comment:
I guess this will just be used for our own analytics?

Reply:
yep, this is just a telemetry bit